marconi-chain-index/src/Marconi/ChainIndex/Indexers/Utxo.hs

```diff
@@ -229,30 +226,38 @@ instance Semigroup (StorableEvent UtxoHandle) where
 instance Monoid (StorableEvent UtxoHandle) where
   mempty = UtxoEvent mempty mempty C.ChainPointAtGenesis
 
--- | The effect of a transaction (or a number of them) on the tx output set.
-data TxOutBalance =
-  TxOutBalance
-    { _tobUnspent :: !(Set C.TxIn)
+-- | The effect of a transaction (or a number of them) on the tx output map.
+data BalanceUtxo =
```

Comment:
Why was the representation changed this way and why was it necessary for this PR? And why the name change (seems to me the old one better fits what it does)?

Reply:
Ah just noticed that it's used in `getUtxoEvents`. I'm good with it.

However, I have an issue with the name. `TxOutBalance` seems clearer.